e2e-tests/cypress/integration/project-application-documents.feature

```diff
@@ -0,0 +1,78 @@
+@testSuite
+Feature: Project Application documents page
```

Comment:
So I feel this is a good start but we don't have enough data with just this one project to really test the full extent of behaviour with the pagination controls.  We don't really explore what the conditional display of the ellipsis for example, or whether the first and last page links are always visible.